README.md

```diff
@@ -1,12 +1,16 @@
-# bpmn-visualization examples
+<h1 align="center">BPMN Visualization Examples</h1> <br>
+<p align="center">
+   <img src="logo.png">
```

Comment:
IMHO, the image doesn't bring any information and will add maintenance.
As an user that wants to integrate the lib, I am looking for usage information, and this example images is only related to the way we layout the various examples pages.

![image](https://user-images.githubusercontent.com/27200110/93759216-175c0300-fc0a-11ea-9ec5-3abf7311b669.png)

Reply:
As a user, I prefer to know what a repository content. The most important part of this one is the demo of examples.
After I know that I can search myself where is the code of them.

So this what I put this picture. Don't need to update it after each example adding, but only if the rendering (format, color...) of the page changes.

Reply:
For the basic library usage, for me this is the library who explain how to do this.

We can add documentation to explain what each example implements.